packages/cgb-scripts/template/src/init.php

```diff
@@ -16,49 +16,50 @@
 /**
  * Enqueue Gutenberg block assets for both frontend + backend.
  *
+ * @uses {wp-blocks} for block type registration & related functions.
+ * @uses {wp-element} for WP Element abstraction — structure of blocks.
+ * @uses {wp-i18n} to internationalize the block's text.
  * @uses {wp-editor} for WP editor styles.
  * @since 1.0.0
  */
 function <% blockNamePHPLower %>_cgb_block_assets() { // phpcs:ignore
 	// Styles.
-	wp_enqueue_style(
+	wp_register_style(
 		'<% blockNamePHPLower %>-cgb-style-css', // Handle.
 		plugins_url( 'dist/blocks.style.build.css', dirname( __FILE__ ) ), // Block style CSS.
-		array( 'wp-editor' ) // Dependency to include the CSS after it.
-		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.style.build.css' ) // Version: File modification time.
+		array( 'wp-editor' ), // Dependency to include the CSS after it.
+		null, // filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.style.build.css' ) // Version: File modification time.
 	);
-}
 
-// Hook: Frontend assets.
-add_action( 'enqueue_block_assets', '<% blockNamePHPLower %>_cgb_block_assets' );
-
-/**
- * Enqueue Gutenberg block assets for backend editor.
- *
- * @uses {wp-blocks} for block type registration & related functions.
- * @uses {wp-element} for WP Element abstraction — structure of blocks.
- * @uses {wp-i18n} to internationalize the block's text.
- * @uses {wp-editor} for WP editor styles.
- * @since 1.0.0
- */
-function <% blockNamePHPLower %>_cgb_editor_assets() { // phpcs:ignore
-	// Scripts.
-	wp_enqueue_script(
+	// Block scripts.
+	wp_register_script(
 		'<% blockNamePHPLower %>-cgb-block-js', // Handle.
 		plugins_url( '/dist/blocks.build.js', dirname( __FILE__ ) ), // Block.build.js: We register the block here. Built with Webpack.
-		array( 'wp-blocks', 'wp-i18n', 'wp-element', 'wp-editor' ), // Dependencies, defined above.
-		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.build.js' ), // Version: File modification time.
+		array( 'wp-blocks', 'wp-i18n', 'wp-element' ), // Dependencies, defined above.
```

Comment:
Why did you remove `'wp-editor'`?

Reply:
This looks like a result of the sync that I did last week with the master branch. I will take care of this right away.

Reply:
Fixed!